code/datums/statistic.dm

```diff
@@ -75,6 +75,36 @@
 	key = "clones"
 	write_to_db = TRUE
 
+/datum/statistic/numeric/people_hung
+	name = "People Hung"
+	key = "hangings"
+	write_to_db = TRUE
+
+/datum/statistic/numeric/self_slaps
+	name = "Total Self-slaps"
+	key = "selfslap"
+	write_to_db = TRUE
+
+/datum/statistic/numeric/ai_doors
+	name = "Doors Opened by AI"
+	key = "AI_DOOR"
```

Comment:
Is there a specific reason everything else is lowercase and only that is uppercase ?

Reply:
This is uppercase purely as a reference to people screaming at the AI.